src/Approximations/decompositions.jl

```diff
@@ -268,6 +268,42 @@ function decompose(S::LazySet{N};
     return CartesianProductArray(result)
 end
 
+
+"""
+    decompose_explicit(S::LazySet{N};
+                       blocks::AbstractVector{Int}=default_block_structure(S, set_type)
+                      )::CartesianProductArray where {N<:Real}
+
+Decompose a high-dimensional set into a Cartesian product of overapproximations
+of the projections over the specified subspaces.
+
+### Input
+
+- `S`           -- set
+- `blocks`      -- (optional, default: [2, …, 2] or [1, …, 1])
+                   block structure - a vector with the size of each block
+
+### Output
+
+A `CartesianProductArray` containing lazy low-dimensional exact
+projections.
+"""
+
+function decompose_explicit(S::LazySet{N};
```

Comment:
I do not understand the name.
Also, I am not sure what you need this function for.